addons/help/XEH_postInit.sqf

```diff
@@ -0,0 +1,43 @@
+//#define DEBUG_MODE_FULL
+#include "script_component.hpp"
+
+if (!hasInterface) exitWith {};
+
+{
+    // create diary, entries added in reverse order
+    private _unit = player;
+    _unit createDiarySubject [QGVAR(docs), "CBA"];
+
+    // add diary for scripted keybinds
+    private _keys = GVAR(keys);
+
+    private _addons = allVariables EGVAR(keybinding,addons);
+    _addons sort true;
+
+    {
+        (EGVAR(keybinding,addons) getVariable _x) params ["_addon", "_addonActions"];
+
+        _keys = _keys + format ["%1:<br/>", _addon];
+
+        {
+            (EGVAR(keybinding,actions) getVariable (_addon + "$" + _x)) params ["_displayName", "", "_keybinds"];
+
+            if (isLocalized _displayName) then {
+                _displayName = localize _displayName;
+            };
+
+            private _keyName = (_keybinds select {_x select 0 > DIK_ESCAPE} apply {_x call CBA_fnc_localizeKey}) joinString "    ";
+
+            _keys = _keys + format ["    %1: <font color='#c48214'>%2</font><br/>", _displayName, _keyName];
+        } forEach _addonActions;
+
+        _keys = _keys + "<br/>";
+    } forEach _addons;
+
+    // delete last line breaks
+    _keys = _keys select [0, count _keys - 10];
+
+    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Help_Keys", _keys]];
+    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Credits", call (uiNamespace getVariable QGVAR(credits))]];
```

Comment:
I think this is failing with large modpack (`count call (uiNamespace getVariable "cba_help_credits")` = 33772)
rpt says ` Unknown entity: ' Accessories</font> by Red Hammer Studios</font><br/><font colo`
that part is exactly 3350 chars into the `call (uiName` string

Reply:
`createDiaryRecord` has a hard limit for chars?

Reply:
It was the & conflicting with the pseudo html of the diary / sqf TEXT.